currency/withdraw.cc.lua

```diff
@@ -0,0 +1,67 @@
+{{/*
+		Simple Currency System By GasInfinity
+        
+        Recommended Trigger: Regex: "^-(withdraw|with)"

```

Comment:
See earlier comments on usage of `\A`. Also don't like `-with`, prefer `withdraw`, `wd`, etc. myself but it's up to you.
```suggestion
        Recommended Trigger: Regex: "\A-(w(ith)?d(raw)?)"
```

Reply:
done